README.md

```diff
@@ -1,6 +1,48 @@
 # Blueprint API
 
-Please visit the infrastructure repo for setting up a local development environment at <https://github.com/CMSgov/blueprint_infra>
+## Local Development Setup
+
+For with access to our current pipeline repository, please visit the infrastructure repo for setting up a local development environment at <https://github.com/CMSgov/blueprint_infra> For others, please follow the steps bellow
+
+This repository is built along with <https://github.com/CMSgov/blueprint_ui> as the backend. They both utilize docker containers, so you will need to have it installed and running to continue.
+
+Follow the UI repository README.md for setting up the local environments first, then you can comback here and run the following commands to start filling out the DB.
+
+The following commands can be run to set up your local envionment
+1) Make a new directory called blueprint
+`mkdir blueprint`
+2) Change directory to blueprint
+`cd blueprint`
+3) Clone the UI and API repositories into the blueprint directory
+`git clone git@github.com:CMSgov/blueprint_ui.git`
+`git clone git@github.com:CMSgov/blueprint_api.git`
+4) Copy the sample docker-compose.yml file into this directory (you might want to edit the file afterwards to use values specific to your system)
+`cp blueprint_api/docker-compose.yml.sample ../docker-compose.yml`
+5) Docker setup
+`docker-compose build`
+`docker-compose run ui npm install`
+`docker-compose up`
+
+6) With the docker container running, change directory to API `cd blueprint_api`
+7) Bash into the repo's docker container (e.g. `docker exec -it blueprint_api_1 bash`) to run the following commands.
+8) Run the current migrations
+`python3 manage.py migrate`
+9) Setup an admin user
+`python3 manage.py createsuperuser`
+
+10) You will next need to add a catalog, sample one can be found at <https://github.com/CMSgov/ars-machine-readable/blob/main/3.1/oscal/json/CMS_ARS_3_1_catalog.json>
```

Comment:
No UI for this now - have to use Postman? Maybe a separate doc (or screenshot) showing required fields.